README.md

```diff
@@ -89,6 +89,10 @@ See the [latest documentation](https://druid.apache.org/docs/latest/) for the do
 
 Make documentation and tutorials updates in [`/docs`](https://github.com/apache/druid/tree/master/docs) using [MarkDown](https://www.markdownguide.org/) and contribute them using a pull request.
 
+The documentation site is built using Docusaurus2. Install Docusaurus2 by running one of the following in `website`: `npm install` or `yarn install`. Then, run `npm start` or `yarn start` in `website/` to see a local preview of your changes.
```

Comment:
Perhaps because of my lack of knowledge (!!!) I ran into an issue starting the docs here because I was still using node 10 (as per https://github.com/apache/druid-website-src/blob/master/README.md) - so perhaps state here which version of node is required to run Docusaurus?

Reply:
Sure, I can update this to add the required Node version.